code/modules/mob/dead/new_player/sprite_accessories/wings.dm

```diff
@@ -223,3 +215,18 @@
 /datum/sprite_accessory/insect_wings/feathery
 	name = "Feathery"
 	icon_state = "feathery"
+
+/datum/sprite_accessory/insect_fluff/colored1
```

Comment:
```suggestion
/datum/sprite_accessory/insect_wings/colored1
```

Reply:
these are actually not wing sprites, I have no idea why they're in the wings sprite accessory file. they are in fact the insect neck fluff

Reply:
just removed them because they're unnecessary anyway